### ISSUE TITLE
JSON RPC is not adhering to JSON-RPC v2.0

### ISSUE THREAD

**Issue by ssoerensen** [badaix/snapcast#161]
While playing around with the JSON-RPC features of snapcast I noticed that it is not possible send a string as request id. to reproduce simply send a id field as a string:

```json
{ "jsonrpc": "2.0", "method": "Server.GetStatus", "id": "1" }
{"error":{"code":-32603,"message":"type must be number, but is string"},"id":null,"jsonrpc":"2.0"}
```
This breaks the usage of some rpc clients. as an example: jsonrpc4j which is sending a string as request id

**Comment by badaix** [comment]
should be fixed in `develop` branch

**Comment by badaix** [comment]
fixed in [0.11.0](https://github.com/badaix/snapcast/releases/tag/v0.11.0-beta-1)
